features/history.yml

```diff
@@ -15,3 +15,5 @@ compat_features:
   - api.History.replaceState
   - api.History.scrollRestoration
   - api.History.state
+  - api.Window.history
```

Comment:
Can you update the description for this feature to mention that this is the entry point?

> "The `window.history` API manipulates the browser session history…